payu_config/archive_scripts/build_intake_ds.py

```diff
@@ -0,0 +1,60 @@
+#!python3
+# Copyright 2024 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+# modules:
+# use:
+#     - /g/data/hh5/public/modules
+# load:
+#     - conda/analysis
+
+from access_nri_intake.source import builders
+from payu.metadata import Metadata
```

Comment:
This doesn't work with `hh5` Payu and for some reason with the following in my `config.yaml` the script couldn't find `access_nri_intake` 

```
use:
    - /g/data/hh5/public/modules
    - /g/data/vk83/modules
load:
    - conda/analysis
    - payu
```

Reply:
These two environments cannot be loaded simultaneously

Reply:
At some point I think we want to add the intake catalog to vk83 to make this easier, but to progress this ill remove the payu dependency